components/restapi/core/blocks.go

```diff
@@ -85,7 +85,7 @@ func blockIssuance() (*apimodels.IssuanceBlockHeaderResponse, error) {
 func sendBlock(c echo.Context) (*apimodels.BlockCreatedResponse, error) {
 	iotaBlock, err := httpserver.ParseRequestByHeader(c, deps.Protocol.CommittedAPI(), iotago.BlockFromBytes(deps.Protocol))
 	if err != nil {
-		return nil, ierrors.Wrapf(err, "failed to parse iotablock")
+		return nil, ierrors.Wrapf(httpserver.ErrInvalidParameter, "failed to parse request: %w", err)
```

Comment:
`ErrInvalidParameter` or `ErrUnsupportedMediaType` is wrapped in the `err` from httpserver, we can wrap `err` directly.

Reply:
lol, you are right! :D
I commented the same in another PR and then I do the same mistake :sweat_smile: :D